dockerfiles/init/modules/openshift/files/scripts/deploy_che.sh

```diff
@@ -344,6 +332,29 @@ else
   CHE_KEYCLOAK_CLIENT__ID=${CHE_KEYCLOAK_CLIENT__ID:-"openshiftio-public"}
 fi
 
+# -------------------------------------------------------------
+# If command == cleanup then delete all openshift objects
+# -------------------------------------------------------------
+if [ "${COMMAND}" == "cleanup" ]; then
```

Comment:
Can you explain why did you move these lines?

Reply:
Thanks for the question Sergio! By my nature I love moving lines around even without any reasons, but not in that case... Here I moved them (with a pleasure I must admit) because in case of `rollupdate ` we call `get_che_pod_config | oc apply -f -` and here before we applied configuration update we need all ENV vars used in `DEFAULT_CHE_CONFIG_FILE_PATH=./che-config` to be inited. And as you can see here before my changes these lines were before `# Setting Keycloak-related environment variables
` section where we init KC related ENVs so that's why.

Reply:
Thanks for the explanation!